[2024-06-04T18:18:43.525+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-04T18:18:43.680+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_ETL_weather.transform_load_weather_data manual__2024-06-04T18:17:56.688371+00:00 [queued]>
[2024-06-04T18:18:43.748+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_ETL_weather.transform_load_weather_data manual__2024-06-04T18:17:56.688371+00:00 [queued]>
[2024-06-04T18:18:43.761+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 4
[2024-06-04T18:18:43.861+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transform_load_weather_data> on 2024-06-04 18:17:56.688371+00:00
[2024-06-04T18:18:45.340+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_ETL_weather', 'transform_load_weather_data', 'manual__2024-06-04T18:17:56.688371+00:00', '--job-id', '98', '--raw', '--subdir', 'DAGS_FOLDER/dag_ETL_weather.py', '--cfg-path', '/tmp/tmp3y7ja955']
[2024-06-04T18:18:45.364+0000] {standard_task_runner.py:91} INFO - Job 98: Subtask transform_load_weather_data
[2024-06-04T18:18:45.399+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=285) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-04T18:18:45.402+0000] {standard_task_runner.py:63} INFO - Started process 287 to run task
[2024-06-04T18:18:45.883+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_ETL_weather.transform_load_weather_data manual__2024-06-04T18:17:56.688371+00:00 [running]> on host 6b740a15ab9e
[2024-06-04T18:18:47.096+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Matheus' AIRFLOW_CTX_DAG_ID='dag_ETL_weather' AIRFLOW_CTX_TASK_ID='transform_load_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2024-06-04T18:17:56.688371+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-04T18:17:56.688371+00:00'
[2024-06-04T18:18:47.104+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-04T18:18:47.557+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-06-04T18:18:48.804+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-06-04T18:18:49.183+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-04T18:18:49.187+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-04T18:18:49.269+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_ETL_weather, task_id=transform_load_weather_data, run_id=manual__2024-06-04T18:17:56.688371+00:00, execution_date=20240604T181756, start_date=20240604T181843, end_date=20240604T181849
[2024-06-04T18:18:49.463+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-04T18:18:49.597+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-04T18:18:49.608+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
